PartyIcons/Configuration.cs

```diff
@@ -18,7 +18,8 @@ public class Configuration : IPluginConfiguration
         public bool TestingMode              = true;
         public bool EasternNamingConvention  = false;
         public bool DisplayRoleInPartyList   = false;
-
+        public bool ShowPlayerStatus = true;
```

Comment:
This is named a little bit too unspecific, 'icon' should be in the name since icon is what it shows.